consensus-engine/src/overlay/random_beacon.rs

```diff
@@ -60,6 +61,15 @@ impl RandomBeaconState {
         Self::Sad { entropy }
     }
 
+    pub fn initial_sad_from_entropy(entropy: [u8; 32]) -> Self {
+        Self::generate_sad(
+            View::new(-1),
+            &Self::Sad {
+                entropy: Box::new(entropy),
+            },
+        )
+    }
+
```

Comment:
Added this call, as nodes should start with an initial sad configuration iirc from the specs. Anyway it is used in the simulations for that purpose at least.